fuzzbench/build_jobs.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Creates build jobs."""
+
+import os
+import subprocess
+import time
+
+from rq.job import Job
+
+
+BASE_TAG = 'gcr.io/fuzzbench'
+
+
+def build_base_images(name):
+    """Builds a Docker image."""
+    image_tag = os.path.join(BASE_TAG, name)
+    return subprocess.run(['docker', 'build',
```

Comment:
How long does it take to build the 3 images you're building with an empty cache?

I think we should pull before build to warm up the cache. (We'll only need the explicit pull until we switched to buildkit. once the images in the registry were built with buildkit and the cache info is inlined in the images, we won't need to explicitly pull, but --cache-from will pull automatically relevant parts.)

Reply:
> How long does it take to build the 3 images you're building with an empty cache?

For `make run-end-to-end-test` command, building run-experiment takes 3 mins, base-image 5 mins, base-builder 1 and base-runner 1 min.

> pull before build to warm up the cache

Does this mean, we first do all pull related tasks and then do build jobs with `--cache-from`?

Reply:
See #558.